index.html

```diff
@@ -25,9 +28,9 @@
         <div class="intro">
             
             <div class="col-12 ">
-                <h1 class="col col-11 col-lg-5 aligne-self-center">I'm Caleb Moses </h1>
```

Comment:
I think for this project you should use HTML and CSS as required instead of Bootstrap